Scala/Scala.sublime-syntax

```diff
@@ -1402,17 +1402,17 @@ contexts:
       scope: punctuation.definition.string.begin.scala
       push:
         - meta_include_prototype: false
-        - meta_scope: string.quoted.triple.scala
-        - match: '{{unicode_char}}'
-          scope: constant.character.escape.scala
+        - meta_scope: meta.string.scala string.quoted.double.block.scala
         - match: '(""")(?!")'
           scope: punctuation.definition.string.end.scala
           pop: true
+        - match: '{{unicode_char}}'
+          scope: constant.character.escape.scala
```

Comment:
Triple strings do not support character escapes. So in other words, we should probably have a test something like this:

```scala
"""\n"""
   ^^ - constant

Reply:
see: line 370